SingularityService/src/main/java/com/hubspot/singularity/data/history/SingularityRequestHistoryPersister.java

```diff
@@ -170,6 +178,13 @@ public void runActionOnPoll() {
         JavaUtils.duration(start)
       );
     } finally {
+      if (persisterSuccess.get()) {
+        LOG.info(
+          "Request Persister: successful move to history ({} so far)",
+          lastPersisterSuccess.incrementAndGet()
```

Comment:
Probably most helpful to set this to something like System.currentTimeMillis() that way we know the last timestamp it ran and can generate an effective 'lag' type of metric off of that. incrementAndGet will just add 1 to the existing value. Logging message would change then too, something like 'finished run on persister at {}' or similar

Reply:
Ok sounds good, I'll make that update.

Should the lastPeristerSuccess be a timestamp rather than an AtomicLong for consumption from InternalSingularityService/sfx?

Reply:
AtomicLong is nice in that we are never competing between threads to read/update it (or if it's inside a lambda and has to be effectively final. It's also hard to update a raw `long` but also bind it in guice as a singleton without something like AtomicLong to wrap it